packages/utilities/src/utilities/markdownconverter/markdownconverter.js

```diff
@@ -0,0 +1,53 @@
+const _htmlTagRegex = /<.*?>/g;
+const _doubleSpaceRegex = /\s{2,}/g;
+const _italicRegex = /[_*](.*?)[_*]/g;
+const _boldRegex = /[_*]{2}(.*?)[_*]{2}/g;
+
+/**
+ *
+ * @param {string} str String to be checked for html tags
+ * @returns {string} String with html tags stripped out
+ */
+const _removeHtmlTags = str => str.replace(_htmlTagRegex, '');
+
+/**
+ *
+ * @param {string} str String to be checked for double spaces
+ * @returns {string} String with html double spaces fixed
+ */
+const _fixDoubleSpaces = str => str.replace(_doubleSpaceRegex, ' ');
+
+/**
+ * Utiltity function for converting markdown into html
```

Comment:
We should probably add more notes on what exactly this tool converts. It doesn't do the full markdown conversion, only particular formatting like italic, bold, etc. Also should mention that any injected HTML is stripped out. Perhaps there should be a flag to allow html to not be stripped?

Reply:
Thanks, I added some more info to the functions description!

Regarding the html stripping flag, do we want to allow that? I thought we wanted to restricted it to just the html tags coming from the markdown converter. Since we will be opening the gates to html usage inside some components text-only areas, allowing for html could result in improper usage of the component I would guess...
But that's just what I assumed, If that's not an issue I could add the flag or remove the html stripping altogether. Let me know your thoughts...

Reply:
right, I guess maybe it strips by default, you have to explicitly set a flag to turn it off? Remember this is a utility that can feasibly be used anywhere. Maybe an adopter would want to use the utility directly in their application.

Reply:
Right, ok I can add that

Reply:
Flag added